apps/web/src/views/NFTs/NFTFilter/NFTFilter.tsx

```diff
@@ -25,11 +26,24 @@ export const NFTFilter = ({
   getCheckboxProps: UseCheckboxGroupReturn["getCheckboxProps"];
 } & ButtonProps) => {
   const color = useColor();
+  const { onOpen, onClose, isOpen } = useDisclosure();
 
   return (
-    <Popover variant="dropdown">
+    <Popover
```

Comment:
You can use `ActionsDropdown` component with some small adjustments like updating PopoverContent `maxWith`